src/xdgmenumaker

```diff
@@ -291,9 +291,11 @@ def icon_full_path(icon):
     # in non-standard directories
     if os.path.exists(icon):
         if max_icon_size:
+	    if os.path.splitext(icon)[1] == ".svg":
```

Comment:
Sorry, the problem with the indentation is that you're using a tab character, instead of spaces as with the rest of the file.